protobufs/nft/service.proto

```diff
@@ -23,19 +23,57 @@ service NFTService {
   }
   rpc MintInvoiceUnpaidNFT(NFTMintInvoiceUnpaidRequest) returns (NFTMintResponse) {
     option (google.api.http) = {
-      post: "/token/mint/invoice/unpaid/{identifier}"
+      post: "/nfts/{identifier}/invoice/unpaid/mint"
```

Comment:
@rdinicut new mint endpoint rest api path

Reply:
we do not care about the paths. That will get updated when we regenerate the api. If the way we call the api changes it will break stuff

Reply:
cool.